### PR TITLE
Migrate to 11ty

### PR DESCRIPTION
**Pull request by leomet07**
Descriptions for every post are needed for SEO 

General site meta description is also needed.

For production, run `npm run build` and host the static output of `_site` on any cdn. You can use `http-server` with ``http-server _site/`` to test.

### CODE REVIEW

File: _data/metadata.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
+	title: "Stuy Linux",
+	url: "https://stuylinux.org/",
+	language: "en",
+	description: "Stuyvesant Linux Club description",
```
Comment (TheEgghead27) [inline _data/metadata.js]: Maybe we could copy the tagline from our [StuyActivities page](https://stuyactivities.org/linux), or our [homepage](https://stuylinux.org/) for consistency

File: _includes/styles/globals.css [hunk 2]
```diff
@@ -11,7 +11,7 @@ body {
 
 a {
 	color: inherit;
-	text-decoration: none;
+	/* text-decoration: none; */
```
Comment (TheEgghead27) [inline _includes/styles/globals.css]: Do we still need this line?

File: content/posts/mac_spoofing.md [hunk 3]
```diff
@@ -20,27 +24,27 @@ dnf install macchanger
...
 leo@artix ~$ ip addr
 1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
     link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
     inet 127.0.0.1/8 scope host lo
        valid_lft forever preferred_lft forever
-    inet6 ::1/128 scope host noprefixroute 
+    inet6 ::1/128 scope host noprefixroute
        valid_lft forever preferred_lft forever
 2: wlp0s20f3: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
     link/ether XX:XX:XX:XX:XX:XX brd ff:ff:ff:ff:ff:ff permaddr WW:WW:WW:WW:WW:WW
```
Comment (TheEgghead27) [inline content/posts/mac_spoofing.md:40]: TBF these are usually local IPs hidden behind NAT
Fair enough though

File: content/posts/minecraft-prismlauncher/minecraft-prismlauncher.md [hunk 4]
```diff
@@ -118,18 +123,17 @@ Then, you will get an `Optifine_blablabla_MOD.jar` in the same folder as your or
...
-![The Version tab. "Add to Minecraft.jar" is highlighted.](/img/posts/minecraft-prismlauncher/add.png)
+![The Version tab. "Add to Minecraft.jar" is highlighted.](/posts/minecraft-prismlauncher/add.png)
 
 You are done, and can now launch with OptiFine.
 
 #### Chem Lab {#chem-lab}
 
-> _honestly at this point fabric jar with lithium and phosphor_
-> \- pcao#2918
+> _honestly at this point fabric jar with lithium and phosphor_ > \- pcao#2918
```
Comment (TheEgghead27) [inline content/posts/minecraft-prismlauncher/minecraft-prismlauncher.md]: Note to self, see how this line came out

**Review by TheEgghead27 — CHANGES_REQUESTED** [review]
Great work! We definitely should think of a good description tag...

**Comment by TheEgghead27** [comment]
Also, while we're at it, do we want to make the migration to a posts submodule that'll be easier to contribute to?
This could help with the embed blurb writing.

**Comment by TheEgghead27** [comment]
Thoughts on importing some new libraries to make header anchors (as used in https://stuylinux.org/posts/minecraft-prismlauncher) to allow for jumping to sections (we could skip `markdown-it-attrs` if the defaults are sane)?
https://syntackle.live/blog/adding-custom-anchors-to-headings-in-markdown-eleventy-3NxBhIJO2OIr4XOj5LKc/

File: _data/metadata.js (hunk 1 above)
Comment (leomet07) [reply _data/metadata.js]: agreed

File: _includes/styles/globals.css (hunk 2 above)
Comment (leomet07) [reply _includes/styles/globals.css]: Deleted the line.

File: _data/metadata.js (hunk 1 above)
Comment (leomet07) [reply _data/metadata.js]: Going to use ``Stuy Linux is Stuyvesant's only initiative to promote the use of Free/Libre Open Source Software (FLOSS) programs and the GNU/Linux projects.`` for now.

**Comment by leomet07** [comment]
> Thoughts on importing some new libraries to make header anchors (as used in https://stuylinux.org/posts/minecraft-prismlauncher) to allow for jumping to sections (we could skip `markdown-it-attrs` if the defaults are sane)? https://syntackle.live/blog/adding-custom-anchors-to-headings-in-markdown-eleventy-3NxBhIJO2OIr4XOj5LKc/

Just followed that great article and added in the anchor tag support! The article links work as expected now, without any weird heading tag behavior.

**Review by TheEgghead27 — APPROVED** [review]
Looks great! Thank you so much :)

**Pull request merged**